lib/digital_public_works/organizations/organization_project.ex

```diff
@@ -0,0 +1,24 @@
+defmodule DigitalPublicWorks.Organizations.OrganizationProject do
+  use Ecto.Schema
+
+  alias DigitalPublicWorks.{Organizations.Organization, Projects.Project}
+
+  @primary_key false
+  @foreign_key_type :binary_id
+
+  schema "organizations_projects" do
+    belongs_to :organization, Organization
+    belongs_to :project, Project
+
+    timestamps()
+  end
+
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> Ecto.Changeset.cast(params, [:organization_id, :project_id])
```

Comment:
Any particular reason for not importing Changeset here?

Reply:
Nope, thanks for pointing this out! I cleaned this up in the affected Ecto schemas.